config/formatting.md

```diff
@@ -14,6 +14,27 @@ All of these examples use the `docker inspect` command, but many other CLI
 commands have a `--format` flag, and many of the CLI command references
 include examples of customizing the output format.
 
+>**Note**
+>
+> Whe using the `--format` flag you need observe your shell environment. In a Posix shell you can run:
+>
+> {% raw %}
+> ```bash
+> docker inspect --format '{{join .Args " , "}}'
+> ```
+> {% endraw %}
+>
+> with single quote. Otherwise, in a Windows shell, e.g Powershell, you need use double quotes like:
+>
+> {% raw %}
+> ```bash
+> docker inspect --format "{{join .Args \" , \"}}"
```

Comment:
Somehow this doesn't work for me either in a PowerShell. 🤔

Reply:
Yeah, I made a mistake with the quotes, can you test again?

Reply:
Thanks, yes it works now!